Isc23 (#409)
aws-samples/aws-hpc-tutorials: content/02-aws-getting-started/03-access-aws.md

##### Step 1

{{% notice warning %}}

> Sandbox is available on May 21st 2023 for the duration of the tutorial. If you would like to run through the labs at a later stage on your own, with your company or institution, please contact us at isctutorial@amazon.com so we can follow-up with you.

Reviewer: ```suggestion
Temporary accounts were available on May 21st 2023 for the duration of the tutorial. If you would like to run through the labs at a later stage on your own, with your company or institution, please contact us at isctutorial@amazon.com so we can follow-up with you.
```
